lib/generators/shopify_app/routes/templates/routes.rb

```diff
@@ -5,3 +5,7 @@
     get 'auth/shopify/callback' => :callback
     get 'logout' => :destroy, :as => :logout
   end
+
+  namespace :webhooks do
+    post ':type' => :receive
+  end
```

Comment:
if you run the routes generator then we unmount the engine and copy the routes that would have been mounted over directly, the same way we will also copy over the controllers and views. It lets you choose between having everything buried in the engine or copied into your app. Quite a few engines work this way (e.g. devise).

It might be worth discussing removing the generators for this though since you can still do this manually and its more of a power used thing.